src/components/coin-row/CoinRow.js

```diff
@@ -47,7 +46,9 @@ export default function CoinRow({
   ...props
 }) {
   const { nativeCurrency, nativeCurrencySymbol } = useAccountSettings();
-
+  const { hiddenCoins, pinnedCoins } = useCoinListEditOptions();
```

Comment:
Since these details are relevant only for the `BalanceCoinRow` and since `BalanceCoinRow` is already calculating these values for the `CoinCheckButton`, we could keep it just as passed in props - wdyt

Reply:
We can, but I don't believe this really matters. I wanted to reduce the number of props chaining